scss/_buttons.scss

```diff
@@ -10,7 +10,8 @@
   white-space: nowrap;
   vertical-align: middle;
   cursor: pointer;
-  @include gradient(#fcfcfc, #eee);
+  background-color: #eee;
```

Comment:
Color literals like `#eee` should only be used in variable declarations; they should be referred to via variable everywhere else.